src/app/components/scan-fingers/scan-fingers.component.html

```diff
@@ -1 +1,37 @@
-<p>scan-fingers works!</p>
+<div
+  class="scanner__wrapper"
+  [ngStyle]="{
+    'border-color': !isReaderConnected
+      ? 'gray'
+      : scanStatus === 'Successful'
+      ? 'green'
+      : 'red'
+  }"
+>
+  <h2 class="scanner__status-title">Fingerprint scan status</h2>
+  <div class="scanner__status">
+    <h2
+      [ngClass]="{
+        'scanner__status--connect-request': true,
+        hidden: !isReaderConnected
+      }"
+      [ngStyle]="{ display: !isReaderConnected ? 'block' : 'none' }"
+    >
+      Please connect<br />fingerprint device
+    </h2>
```

Comment:
it seems we are missing a case when the device is connected, we would show a kind of message as follow `Please scan your fingers`, no?

Reply:
@hoangngk Yes. Updated